framework/configuration/modules/different-module-types.rst

```diff
@@ -26,34 +26,58 @@ For every customer installation a customer module exists. A customer module conf
 A customer module defines which optional modules (marketing modules) are installed. Also customer specific adjustments
 which only belong to the customer are done in customer modules.
 
-Customer modules can be found in: ``path/to/nice2/customer/*``
+Customer modules can be found in: ``customer/*``
 
-Between Modules
-^^^^^^^^^^^^^^^
+Intermediate Modules
+^^^^^^^^^^^^^^^^^^^^
 
-A between module is always needed if a feature or functionality should be added **automatically** if two or more specific
-modules are installed and this new feature should **not be licenced separately**. If the feature was licenced separately
-it could just be an optional module.
+A intermediate module is always needed if a feature or functionality should be added **automatically**
+[#f1]_ if two or more specific modules are installed and this new feature should **not be licensed
+separately**. If the feature is licensed separately, an optional module must be created instead.
```

Comment:
That's not true anymore. There are intermediate modules which are licensed separately and a certain optional module can contain multiple items that are licensed separately. I think you can simply remove the part about the licenses here.